cmd/postgres_exporter/server.go

```diff
@@ -128,6 +131,17 @@ func (s *Server) Scrape(ch chan<- prometheus.Metric, disableSettingsMetrics bool
 		err = fmt.Errorf("queryNamespaceMappings returned %d errors", len(errMap))
 	}
 
+	{
+		pgdb := collector.NewPGDatabaseCollector()
```

Comment:
Rather than put every collector in the `main`, we should probably do what we do in the node_exporter.

So in `collector/pg_database.go`

```
func init() {
  registerCollector("pg_database", defaultDisabled, NewPGDatabaseCollector)
}
```